applications/asset_tracker_v2/tests/nrf_cloud_codec_mocked_cjson/src/nrf_cloud_codec_mocked_test.c

```diff
@@ -478,7 +478,6 @@ void test_enc_batch_data_bat_no_data_obj(void)
 void test_enc_batch_data_gnss_no_data_obj(void)
 {
 	struct cloud_data_gnss gnss_buf = {
-		.format = CLOUD_CODEC_GNSS_FORMAT_NMEA,
```

Comment:
`CLOUD_CODEC_GNSS_FORMAT_NMEA` is set here but the actual NMEA string is not there. In this PR, we have only PVT format so format parameter is removed. So I guess I can just remove this line?
@maxd-nordic

Reply:
Discussed that this is fine.